app/controllers/customers_controller.rb

```diff
@@ -20,25 +20,32 @@ def new
   # GET /customers/1/edit
   def edit
   end
-
   # POST /customers
   # POST /customers.json
-  def create
-    @customer = Customer.new(customer_params)
 
+  def create
+    if phone_exist(params[:customer][:phone_number])
+      @customer = Customer.find_by(phone_number: params[:customer][:phone_number])
+      flag = true
```

Comment:
@kalgibhatt-btc What is `flag`? It is not easy to understand what it does, should be named properly.